### PR TITLE
Add hosted plugin docs

### LINKED ISSUE #29403: Update Jamf docs to mention hosted Jamf plugin
## Applies To

[Jamf Integration](https://goteleport.com/docs/access-controls/device-trust/jamf-integration/?scope=enterprise)

## Details

In Jamf Integration doc, mention that we support hosted Jamf plugin in cloud and users have option to both run Jamf as a seperate service or as a hosted plugin.

### LINKED ISSUE #32847: Create Docs for Hosted Okta Integration
The docs do not mention the hosted Okta option under Management > Integrations > Enroll New Integration > Okta

https://goteleport.com/docs/application-access/okta/

_Moved from: https://github.com/gravitational/docs/issues/402_

### PR DESCRIPTION
**Pull request by ptgott**
See #29979
Closes #32847
Closes #29403

For the most part, the documentation for the Teleport Access Request plugins assumes that a user is self-hosting their plugins, and contains information for installing and configuring the plugin that is not relevant to a user of hosted plugins.

This change edits the menu page of the Access Request plugin subsection of the docs to add instructions for enrolling hosted Access Request plugins. It also adds a note to the top of each guide for an Access Request plugin with a hosted counterpart that a user can enable the hosted version of that plugin. This anticipates a separate change to add more detailed instructions to the hosted Access Request plugin enrollment workflow within the Web UI.

Currently, the Opsgenie guide is the only one for an Access Request plugin that has no self-hosted counterpart. To keep all docs for specific hosted Access Request plugins within the Web UI, this change removes that guide and adds a redirect.

**Reviewer note:** This PR depends on Web UI changes made in https://github.com/gravitational/teleport.e/pull/2116.

### CODE REVIEW

**Comment by ptgott** [comment]
Converting to draft while I add information for Jamf and Okta

**Review by zmb3 — COMMENTED** [review]
Aren't we losing some information from the Opsgenie guide if we just remove it?

**Comment by ptgott** [comment]
@zmb3 I'm moving the relevant information into the Web UI in this PR, which is going through design review: https://github.com/gravitational/teleport.e/pull/2116

I'll make it more explicit that this PR depends on that one and move this into "Draft".

**Comment by ptgott** [comment]
I'm restoring the OpsGenie plugin docs in 86cd67ba5bbca0e4e872ed2a374b64f43cc7c392. 

Users who want to know about the OpsGenie plugin via organic search can find the docs they need, and this doesn't interfere with user journeys that begin at the "Access Request Plugins" section menu page, which makes it explicit that OpsGenie is a hosted plugin.

**Review by zmb3 — APPROVED** [review]

**Comment by ptgott** [comment]
@lsgunn-teleport Would you have time to give this one a look?

**Review by stevenGravy — APPROVED** [review]

File: docs/pages/application-access/okta/guide.mdx [hunk 1]
```diff
@@ -14,6 +14,12 @@ with user groups can be managed by Teleport's RBAC along with access requests.
 This guide will help you set up the Okta Service and provide some basic
 configurations to enable it.
 
+<Details title="This integration is hosted on Teleport Cloud" open={false}>
+
+(!docs/pages/includes/plugins/enroll.mdx!)
```
Comment (ibeckermayer) [inline docs/pages/application-access/okta/guide.mdx:19]:
```suggestion
(!docs/pages/includes/plugins/enroll.mdx name="the Okta integration"!)
```

**Review by ibeckermayer — APPROVED** [review]

**Pull request merged**